src/libpmemobj/heap.c

```diff
@@ -815,11 +851,19 @@ heap_populate_bucket(struct palloc_heap *heap, struct bucket *bucket)
 {
 	struct heap_rt *h = heap->rt;
 
+	unsigned zone_id = h->nzones; /* first not reclaimed zone */
+	for (zone_id = 0; zone_id < h->nzones; ++zone_id) {
```

Comment:
Is there a reason why `zone_id` is set to `h->nzones` if its value is set to 0 in the for loop immediately after? It looks a bit suspicious to me.